gen/nvml/init.go

```diff
@@ -115,6 +117,20 @@ func (pis ProcessInfo_v1Slice) ToProcessInfoSlice() []ProcessInfo {
 	return newInfos
 }
 
+func (pis ProcessInfo_v2Slice) ToProcessInfoSlice() []ProcessInfo {
+	var newInfos []ProcessInfo
+	for _, pi := range pis {
+		info := ProcessInfo{
+			Pid:               pi.Pid,
+			UsedGpuMemory:     pi.UsedGpuMemory,
+			GpuInstanceId:     pi.GpuInstanceId,
+			ComputeInstanceId: pi.ComputeInstanceId,
```

Comment:
Looking at this, the two types (`_v2` and unversioned) are the same struct as of this change?

Reply:
Yeah, I don't understand what they are trying to accomplish here. Maybe it was a mistake and the "v3" struct was supposed to have more fields. The comment seems to indicate so.